packages/frontend/src/components/campaigns/rewards/index.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
+
+import { Typography } from "@/src/ui/typography";
+import styles from "./styles.module.css";
```

Comment:
Same here, and for the others below.

Reply:
Addressed in 1cc81e200b9035b50c9a7c9f8a5721dbae9926bd